src/middlewared/middlewared/plugins/vm.py

```diff
@@ -563,6 +563,14 @@ def identify_hypervisor(self):
                 vnc_devices.append(vnc)
         return vnc_devices
 
+    @accepts()
+    def get_vnc_ifaces(self):
```

Comment:
If I were a user and saw that in /api/docs I would expect `vm.get_vnc_ifaces` to return an interface list: `["em0", "igb0"]`.

Would you agree its more obvious to use `get_vnc_ips` ?